lib/icinga/clusterevents-check.cpp

```diff
@@ -75,6 +75,31 @@ void ClusterEvents::EnqueueCheck(const MessageOrigin::Ptr& origin, const Diction
 	}
 }
 
+static void sendEventExecuteCommand(const Dictionary::Ptr& params, const long& exit, const String& output,
+									const double& start, const double& end, const ApiListener::Ptr& listener, const MessageOrigin::Ptr& origin,
+									const Endpoint::Ptr& sourceEndpoint) {
```

Comment:
Please adapt [our style](https://github.com/Icinga/icinga2/blob/9f57e895f153409fa9f466efd8343efd9e7c9fd2/lib/icingadb/icingadb-objects.cpp#L382-L384) for functions with many params (but indent only with one tab!).